### PR TITLE
[ #3 ]  Add Node.js code samples for each feature described in the README

### PR DESCRIPTION
**Pull request by Mephalich**
Added nodejs folder with tests and readme.md.

### CODE REVIEW

**Comment by SuzeShardlow** [comment]
Closes #3 when approved and merged.

File: nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+4. Hashes: HRANDFIELD Command
+   {"10":"Szechuan Shredded Beef","11":"King Prawns with Cashew Nuts","12":"Roast Duck with Plum Sauce","13":"Grilled Lemon Chicken","14":"Chicken with Pineapple","15":"Beef with Broccoli","16":"Malaysian Chicken Curry","17":"House Special Chow Mein","18":"Mushroom Fried Rice","19":"Fried Mushrooms with Black Bean Sauce","20":"Singapore Fried Rice"}
+   15
+   {"14":"Chicken with Pineapple","16":"Malaysian Chicken Curry"}
+   {"13":"Grilled Lemon Chicken","17":"House Special Chow Mein"}
+5. Sets: SMISMEMBER Command
+   [ '3', '5', '19', '27', '33', '56' ]
+   [ false, true, false, true, false, true ]
+```
+``Note - results might be abit offdue to random functions used``
```
Comment (simonprickett) [inline nodejs/README.md:70]:
```suggestion
``Note - results might be a bit off due to random functions used``
```

File: nodejs/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# Examples using 'node-redis' client
+
+This repository contains example data in nodejs using node-redis client.
+
+## Setup and Start
+
+### Sample Data
+Use the main readme.md for sample data load. Some examples are using preloaded keys.
```
Comment (simonprickett) [inline nodejs/README.md:8]:
```suggestion
Use the main README.md for sample data load. Some examples are using preloaded keys.
```

File: nodejs/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+```bash
+docker exec -it hitc_redis sh /sampledata/load_all.sh
+```
+
+### Reload Sample Data
+Some tests are modifying the Sample Data so the next run will not represent the same dataset for tests. To reload everything use the next commands:
```
Comment (simonprickett) [inline nodejs/README.md:14]:
```suggestion
Some tests modify the Sample Data so the next run will not start with the same dataset. To reload everything use the following commands:
```

File: nodejs/index.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
+import { createClient } from 'redis'
+
+const fnc = async() => {
+    // Lets connect
+    const client = createClient({ url:'redis://localhost:6379' });
```
Comment (simonprickett) [inline nodejs/index.js:5]:
```suggestion
    const client = createClient({ url:'redis://localhost:6379' })
```
Just removing `;` for consistency with the rest of the script.

File: nodejs/index.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    client.on('error', (err) => { console.log('Redis Client Error: ', err); process.exit(2)});
```
Comment (simonprickett) [inline nodejs/index.js:6]:
```suggestion
    client.on('error', (err) => { console.log('Redis Client Error: ', err); process.exit(2)})
```

File: nodejs/index.js [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+    await client.connect();
```
Comment (simonprickett) [inline nodejs/index.js:8]:
```suggestion
    await client.connect()
```

**Review by simonprickett — APPROVED** [review]
@SuzeShardlow  -- this is good to merge thanks @Mephalich !

**Comment by SuzeShardlow** [comment]
Thanks @Mephalich, I've given you the `Hacktoberfest 2022` role in Discord :)

**Pull request merged**